### DISCUSSION TITLE
UC 4 Post Interaction Backend

### DISCUSSION THREAD

**Discussion by hueyee** [SparkSketch-Team/SparkSketch#45] [General]
For the most part I think it's done. Other items in the UC can be achieved in Frontend. Let me know if you need any more server functionality @acsmaker-code

**Comment by acsmaker-code** [comment]
Ok I will get working now that I actually have free time

**Comment by hueyee** [comment]
@acsmaker-code fixed Add Comments. Here are things that don't work: no ability to remove likes. Comments also do not display.

**Reply by hueyee** [reply]
You will need to run `dotnet ef database update` before any work on your end is done.

**Reply by acsmaker-code** [reply]
when I do this in vscode it says dotnet ef does not  exist - where am I supposed to run this?
